src/Home.tsx

```diff
@@ -1,14 +0,0 @@
-import * as React from 'react'
-import { Header } from './Header'
-import RouteResolver from './RouteResolver'
-
-const Home = () => {
-  return (
-    <div>
-      <Header />
-      <RouteResolver />
```

Comment:
Didn't need another "RouteResolver" in the tree. Since this is just rendering a Header, let's just remove this entire component.